mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js

```diff
@@ -100,7 +100,7 @@ OpenLayers.Lang.de = OpenLayers.Util.extend(OpenLayers.Lang.de, {
     /* GEOR_cswbrowser.js strings */
     "NAME layer": "Layer ${NAME}",
     "Metadata without a name": "Namenlose Metadaten",
-    "The getDomain CSW query failed": "Die CSW getDomain Anfrage ist gescheitert",
+    "The request to GeoNetwork 4 aggregations failed": "Die Geonetwork 4 aggregations Anfrage ist gescheitert",

```

Comment:
Isn't there a capital N in GeoNetwork ?

Reply:
```suggestion
    "The request to GeoNetwork 4 aggregations failed": "Die GeoNetwork 4 aggregations Anfrage ist gescheitert",

```

Reply:
@fvanderbiest thanks, yes you're right, you can make suggestions so I can merge them instantly.